ui/Screen/Org/Fund.svelte

```diff
@@ -1,5 +1,6 @@
 <script>
-  import { transactions as store } from "../../src/transaction.ts";
+  import { transactions as txStore } from "../../src/transaction.ts";
+  import { org as orgStore } from "../../src/org.ts";
   import { Wallet, Remote } from "../../DesignSystem/Component";
 
   export let params = null;
```

Comment:
It'd be nice to move the `orgTransactions` function into either `org.ts` or `transaction.ts` so it can be typesafe & testable.

Reply:
Yes good catch. This has been done here: https://github.com/radicle-dev/radicle-upstream/pull/691/commits/3899cd8c8fdda44ba868d618a78a9a20c485f9cc
I also think i'll improve it by not having it match a string, as you mentioned in an other review.